patches/chrome-browser-ui-views-frame-browser_frame_view_win.cc.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/chrome/browser/ui/views/frame/browser_frame_view_win.cc b/chrome/browser/ui/views/frame/browser_frame_view_win.cc
-index bd59ccd1b178450c7eba720ea8b235f24416a15e..3b2be5646116d60f79baf3e968aa164245dde937 100644
+index bd59ccd1b178450c7eba720ea8b235f24416a15e..40d21c79c71f8fb76dcaa8910ebd011db0443d19 100644
 --- a/chrome/browser/ui/views/frame/browser_frame_view_win.cc
 +++ b/chrome/browser/ui/views/frame/browser_frame_view_win.cc
-@@ -547,6 +547,7 @@ int BrowserFrameViewWin::TopAreaHeight(bool restored) const {
+@@ -488,6 +488,7 @@ int BrowserFrameViewWin::FrameTopBorderThickness(bool restored) const {
+       // default. When maximized, the OS sizes the window such that the border
+       // extends beyond the screen edges. In that case, we must return the
+       // default value.
++      BRAVE_BROWSER_FRAME_VIEW_WIN_FRAME_TOP_BORDER_THICKNESS
```

Comment:
Any chance we can subclass `BrowserFrameViewWin`, make `FrameTopBorderThickness` and override it?

Reply:
I'll look into it. The existing patch in this file does basically the same thing that this one does: mimics the early return that upstream has for its 2023 refresh. We should probably change techniques for both at the same time. Would it be OK to go with this approach for now (we'll want to uplift) and create an issue to explore a subclassing approach?

Reply:
Sure, let's create a follow up issue, if you don't mind.

Reply:
@emerick has pointed out that a subclass already exists, so I'll take another look at changing the strategy in this PR.

Reply:
https://github.com/brave/brave-browser/issues/35980